### ISSUE TITLE
[a11y] Loans cannot be made from preparations

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1381]
**Describe the bug**
You cannot create a Loan after selecting preparations.

**To Reproduce**
1. Go to Interactions
2. Create a Loan
3. Choose a record set
4. Select some number of preparations
5. Click apply
6. See no change

**Expected behavior**
A loan should be created

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/166500496-2369e834-b80e-4803-84ef-510a2d76039d.png)

**Desktop (please complete the following information):**
macOS, Edge

**Comment by maxxxxxdlp** [comment]
Duplicate of #1275
